### PR TITLE
REL-4265: Igrins2 fixes

### PR DESCRIPTION
**Pull request by cquiroz**
This PR includes 3 changes for Igrins2:
* Remove the SlitViewingCamera porperty
* Allow the creation of an Igrins2-iterator
* Fix a UI bug when changing the exposure time

### CODE REVIEW

File: bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/igrins2/SeqConfigIgrins2CB.java [hunk 1]
```diff
@@ -21,6 +36,12 @@ public Object clone() {
     @Override
     protected void thisApplyNext(final IConfig config, final IConfig prevFull) {
         super.thisApplyNext(config, prevFull);
+
+        copyToObserve(config, Igrins2.EXPOSURE_TIME_PROP());
+
+        config.putParameter(SeqConfigNames.INSTRUMENT_CONFIG_NAME,
+                StringParameter.getInstance(InstConstants.INSTRUMENT_NAME_PROP,
+                        Igrins2.INSTRUMENT_NAME_PROP()));
```
Comment (swalker2m) [inline bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/igrins2/SeqConfigIgrins2CB.java:44]: I can't remember the details, but somehow this happens for `Flamingos2` without having to explicitly put the parameter in the config builder.

**Review by swalker2m — APPROVED** [review]
Seems good to me.  :+1:

File: bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/igrins2/SeqConfigIgrins2CB.java (hunk 1 above)
Comment (cquiroz) [reply bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/igrins2/SeqConfigIgrins2CB.java:44]: This was taken from Ghost, and the same was done for GPI but maybe I should remove it?

Comment (swalker2m) [reply bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/igrins2/SeqConfigIgrins2CB.java:44]: It's not worth spending any time on it.  If it works then :+1:

**Pull request merged**